weather/tests/test_data_funcs.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+from data_funcs import isfloat
+from data_funcs import to_float
+
+
+# ===================================================================
+# Tests for isfloat()
+# ===================================================================
+
```

Comment:
Are you going to use pytest? If so, you can use pytest.mark.parametrize to condense each test case into one function.